interface/src/app/analyze/analysis/sampleBin.tpl.html

```diff
@@ -1,10 +1,9 @@
-<svg id="sample-bin" width="100" height="60" ng-click="show()">
-  <image x="27" y="6" width="45" height="45"
-      xlink:href="static/assets/testtube-expt2.svg">
-  </image>
-  <g ng-if="sb.heatmapData.samples.length">
-    <text text-anchor="left" x="28" y="43">&times;</text>
-    <text text-anchor="left" x="44" y="43">{{
-      sb.heatmapData.samples.length }}</text>
-  </g>
-</svg>
+<li ui-sref-active="active" class="sample-bin">
+  <a href ui-sref="analysis-detail">
```

Comment:
Thank you for the links. According to:
https://github.com/angular-ui/ui-router/wiki/Quick-Reference#ui-sref
`href` will be added automatically when the template html is compiled into the actual HTML. YOu can probably test it by removing `href` from the code, if it works, then it is safe to get rid of it.